### PR TITLE
Added unit tests for hooks module

### PR DESCRIPTION
**Pull request by Stranger6667**

### CODE REVIEW

**Comment by Lukasa** [comment]
This also seems reasonable to me, but @Stranger6667 is sat right next to me. ;) @sigmavirus24/@kennethreitz, mind doing an extra review for me? This is +1 from me.

**Comment by ryandebruyn** [comment]
Hello folks seems somehow I got placed on this email alias By mistake.
Anyway I can be removed.

Thank you
-Ryan

**Comment by Lukasa** [comment]
@ryandebruyn This is managed by GitHub, it's not an email alias. You'll need to take some action to remove yourself via GitHub.

File: requests/hooks.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 """
 HOOKS = ['response']
 
+
```
Comment (sigmavirus24) [inline requests/hooks.py]: I know this is pep8, but it's unnecessary and is not a unit test. ;)

File: tests/test_hooks.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+import pytest
+
+from requests.hooks import dispatch_hook, default_hooks
```
Comment (sigmavirus24) [inline tests/test_hooks.py]: I would rather just import `hooks` from requests instead of the two functions.

**Comment by sigmavirus24** [comment]
@ryandebruyn you may be "Watching" this repository in which case only you can fix that by visiting the project page while logged in and "un-watching" it.

**Comment by Stranger6667** [comment]
Hello, @sigmavirus24 @Lukasa !
I've updated this PR :)

File: tests/test_hooks.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    return value[1:]
+
+
+@pytest.mark.parametrize(
+    'hooks_list, result', (
+        (hook, 'ata'),
+        ([hook, lambda x: None, hook], 'ta'),
+    )
+)
+def test_hooks(hooks_list, result):
```
Comment (Lukasa) [inline tests/test_hooks.py:17]: So, this test is now pretty opaque: looking at the parametrized results don't make it enormously clear what it's supposed to be testing. A docstring or comment that explains how this test should function would be good.

**Comment by Lukasa** [comment]
I have one small note. =)

File: tests/test_hooks.py (hunk 3 above)
Comment (kennethreitz) [inline tests/test_hooks.py:17]: +1

**Comment by kennethreitz** [comment]
@Stranger6667 I don't like having tons of pull requests open, so I'm merging this. If you wouldn't mind, it would be appreciated if you opened another PR that addresses the notes @Lukasa left on your test.

**Pull request merged**